### PR TITLE
[5.8] Added some tests

### PR DESCRIPTION
**Pull request by imanghafoori1**
These test try to cover edge cases. Specially for unserialized objects which php 7.2 is_object() behaves differently on.

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Thanks for the PR. Is there any chance you could send this to 5.7 please?

**Comment by imanghafoori1** [comment]
> Thanks for the PR. Is there any chance you could send this to 5.7 please?

of cource why not

File: tests/Support/SupportArrTest.php [hunk 1]
```diff
@@ -682,5 +682,18 @@ public function testWrap()
...
+        $this->assertEquals([null, null], Arr::wrap([null, null]));
+        $this->assertEquals([''], Arr::wrap(''));
+        $this->assertEquals([''], Arr::wrap(['']));
+        $this->assertEquals([false], Arr::wrap(false));
+        $this->assertEquals([false], Arr::wrap([false]));
+        $this->assertEquals([0], Arr::wrap(0));
+
+        $obj = new stdClass;
+        $obj->value = 'a';
+        $obj = unserialize(serialize($obj));
```
Comment (jmarcher) [inline tests/Support/SupportArrTest.php]: Why is this line needed?

Comment (imanghafoori1) [reply tests/Support/SupportArrTest.php]: Following https://github.com/laravel/framework/pull/26966 failing PR, we decided to add tests preventing that mistake.
Also check : https://secure.php.net/manual/en/function.is-object.php

**Comment by GrahamCampbell** [comment]
Replaced by https://github.com/laravel/framework/pull/26974.
